proto/cosmos/slashing/genesis.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+package cosmos.slashing;
+
+option go_package            = "github.com/cosmos/cosmos-sdk/x/slashing/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/slashing/slashing.proto";
+
+// GenesisState - all slashing state that must be provided at genesis
+message GenesisState {
+  Params params = 1 [
+    (gogoproto.casttype) = "Params",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated SigningInfo signing_info = 2 [
+    (gogoproto.moretags) = "yaml:\"signing_info\"",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated ValidatorMissedBlocks missed_blocks = 3 [
+    (gogoproto.moretags) = "yaml:\"missed_blocks\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+// SigningInfo stores validator signing info of corresponding address
+message SigningInfo {
+  string address = 1;
+  ValidatorSigningInfo validator_signing_info = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"validator_signing_info\""
+  ];
+}
+
+// ValidatorMissedBlocks contains array of missed blocks of corresponding address
+message ValidatorMissedBlocks {
+  string address = 1;
+  repeated MissedBlocks missed_blocks = 2 [
```

Comment:
ah sorry @anilCSE, you were totally right about https://github.com/cosmos/cosmos-sdk/pull/6881#discussion_r463525966! Let's stick to what you said there, makes more sense.

```suggestion
  repeated MissedBlock missed_blocks = 2 [
```